.github/workflows/deploy-main.yml

```diff
@@ -34,12 +34,12 @@ jobs: # We need two separate builds as each is going to a different ECR repo
     needs: build-staging
     with:
       deploy_to_environment: staging
-      lambda: staging-ep-event-delivery
+      lambda: staging-ep-indexer
```

Comment:
why mention 'indexer' in here if this is no longer the indexer lambda?

Reply:
Fixed